backend/FwLite/MiniLcm/IMiniLcmWriteApi.cs

```diff
@@ -12,7 +12,9 @@ Task<WritingSystem> UpdateWritingSystem(WritingSystemId id,
         WritingSystemType type,
         UpdateObjectInput<WritingSystem> update);
 
-    Task CreatePartOfSpeech(PartOfSpeech partOfSpeech);
+    Task CreatePartOfSpeech(PartOfSpeech partOfSpeech); // TODO: return Task<PartOfSpeech> instead?
```

Comment:
Should CreatePartOfSpeech return the created object the way CreateEntry does? For now I've left the API shape as-is, but this is an inconsistency I'd like to discuss and resolve.

Reply:
Done in commit 3f705dd8.